### PR TITLE
Dev/android/participation

### PR DESCRIPTION
**Pull request by AlicanM**
I will implement the event participations feature to the mobile app, details are in [issue](https://github.com/bounswe/bounswe2022group7/issues/564)

### CODE REVIEW

**Comment by atillaturkmen** [comment]
@AlicanM 
The feature works correctly on the branch, I will merge this pull request after merge conflicts are resolved.

**Review by atillaturkmen — CHANGES_REQUESTED** [review]
@AlicanM
The feature works correctly on the branch, I will merge this pull request after merge conflicts are resolved.

**Comment by AlicanM** [comment]
> @AlicanM The feature works correctly on the branch, I will merge this pull request after merge conflicts are resolved.

I resolved the conflicts. @atillaturkmen

**Pull request merged**
